src/utils/hardCodedSizes/index.js

```diff
@@ -1,3 +1,5 @@
-export const mobileMediaQuerySize = 640
+export const minimumScreenWidth = 320
+export const smallScreenWidth = 640
+export const largeScreenWidth = 992
```

Comment:
I'd like to remove these at some point and only use tachyons, but it will require more work and this works for now